src/main/resources/application.properties

```
@@ -1,12 +1,12 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/book_store?serverTimeZone=UTC
-spring.datasource.username=root
-spring.datasource.password=Sql!2023Java
+spring.config.import=optional:file:.env[.properties]
+spring.datasource.url=jdbc:mysql://localhost:${MYSQLDB_DOCKER_PORT}/\
```

```suggestion
spring.datasource.url=jdbc:mysql://localhost:${MYSQLDB_DOCKER_PORT}/\
```
Is everything fine here? `RT}/\`